lib/realtime/server.ex

```diff
@@ -204,6 +239,19 @@ defmodule Realtime.Server do
     VehicleOrGhost.find_by(vehicles_to_search, search_params)
   end
 
+  def lookup({table, {:limited_search, search_params}}) do
+    logged_in_vehicles = lookup({table, :logged_in_vehicles})
+
+    vehicles_to_search =
+      if Map.get(search_params, :include_logged_out_vehicles, false) do
+        logged_in_vehicles ++ lookup({table, :logged_out_vehicles})
+      else
+        logged_in_vehicles
+      end
```

Comment:
**suggestion (non-blocking):** I think we now have this same basic logic repeated a few times in this file, it might be worth splitting out into a helper function.